contracts/examples/NonMerklizedIdentityExample.sol

```diff
@@ -0,0 +1,163 @@
+    using IdentityLib for IdentityLib.Data;
+
+    // These variables are used to create core claims
+    // They represent the type of claim
+    string private jsonldSchemaURL;
+    uint256 private jsonldSchemaHash;
+    string private jsonSchemaURL;
+    string private credentialType;
+
+    // countOfIssuedClaims count of issued claims for incrementing revocation nonce for new claims
+    uint64 private countOfIssuedClaims = 0;
+   
+    // ClaimInfo represents a claim and its metadata
+    struct ClaimInfo {
+        // metadata
+        string jsonldSchemaURL;
+        uint256 jsonldSchemaHash;
+        string jsonSchemaURL;
+        string credentialType;
+        // data
+        uint256[8] claim;
+    }
+
+    // claimsMap claims storage
+    mapping(uint256 => mapping(string => ClaimInfo)) private claimsMap;
+
+    // saveClaim save a claim to storage
+    function saveClaim(
+        uint256 _id,
+        string memory _uuid,
+        uint256[8] memory _claim
+    ) private {
+        claimsMap[_id][_uuid] = ClaimInfo({
+            jsonldSchemaURL: jsonldSchemaURL,
+            jsonldSchemaHash: jsonldSchemaHash,
+            jsonSchemaURL: jsonSchemaURL,
+            credentialType: credentialType,
+            claim: _claim
+        }); 
+        countOfIssuedClaims++;
+    }
+
+    /**
+     * @dev Get user claim by user id and uuid
+     * @param _userId - user id
+     * @param _uuid - uuid of the claim that was used for issuing claim
+     */
+    function getUserClaim(uint256 _userId, string memory _uuid) public view returns (ClaimInfo memory) {
+        return claimsMap[_userId][_uuid];
+    }
+
+    /**
+     * @dev Set schema
+     * @param _jsonldSchemaURL  - JSONLD schema url
+     * @param _jsonldSchemaHash - JSONLD schema hash. Use this code snippet to calculate hash: https://go.dev/play/p/3id7HAhf-Wi 
+     * @param _jsonSchemaURL  - JSON schema url
+     * @param _credentialType - credential type
+     */
+    function setSchema(
+            string memory _jsonldSchemaURL, 
+            uint256 _jsonldSchemaHash,
+            string memory _jsonSchemaURL,
+            string memory _credentialType
+        ) public onlyOwner {
+        jsonldSchemaURL = _jsonldSchemaURL;
+        jsonldSchemaHash = _jsonldSchemaHash;
+        jsonSchemaURL = _jsonSchemaURL;
+        credentialType = _credentialType;
+    }
+
+
+    // This empty reserved space is put in place to allow future versions
+    // of the State contract to inherit from other contracts without a risk of
+    // breaking the storage layout. This is necessary because the parent contracts in the
+    // future may introduce some storage variables, which are placed before the State
+    // contract's storage variables.
+    // (see https://docs.openzeppelin.com/upgrades-plugins/1.x/writing-upgradeable#storage-gaps)
+    // slither-disable-next-line shadowing-state
+    // slither-disable-next-line unused-state
+    uint256[500] private __gap;
```

Comment:
remove this _gap, it should be before storage vars

Reply:
done